contracts/modules/MerkleDropMinter.sol

```diff
@@ -26,6 +26,14 @@ contract MerkleDropMinter is IMerkleDropMinter, BaseMinter {
      */
     mapping(address => mapping(uint128 => EditionMintData)) internal _editionMintData;
 
+    /**
+     * @dev The number of mints for each account.
+     *      Maps `edition` => `mintId` => `address` => value.
+     *      We will simply store a uint256 for every account, to keep the Merkle tree
+     *      simple, so that it is compatible with 3rd party allowlist services like Lanyard.
+     */
+    mapping(address => mapping(uint128 => mapping(address => uint256))) internal _mintCounts;
```

Comment:
Don't you need to add the logic to update this during `mint()` or are you planning to do that in a follow up?